Add distance_calculator

Mathdallas-code opened the pull request:
I'd like to add a distance calculation program i made

Mathdallas-code committed:
Add files via upload

Mathdallas-code commented:
@souravjain540 please review

jacobliu000 reviewed (changes requested):
The code works for positive integers but doesn't accept negative points (ex: (-2,0)). In addition, there should be a way to break out of the while loop, such as inputting a sentinel value.

Mathdallas-code commented:
> The code works for positive integers but doesn't accept negative points (ex: (-2,0)). In addition, there should be a way to break out of the while loop, such as inputting a sentinel value.

Ok I will try to fix it, and thanks for telling 👍 

Edit: I fixed the bug and it was actually really easy. I just replaced every `.isdigit()` with `.isalpha()`

Mathdallas-code committed:
Fixed bugs and added feature

Fixed a bug where negative numbers were not allowed and added a quit features

jacobliu000 reviewed (approved):
Looks good!

Mathdallas-code committed:
Fixed floating point bug

Made the calculator compatible with floating(decimal)numbers

Mathdallas-code commented:
> Looks good!

Thanks! I also fixed a bug where decimal numbers were incompatible.

Mathdallas-code committed:
Added better input validation
